src/domains/observability/env/prod/terraform.tfvars

```diff
@@ -259,13 +259,31 @@ ehns_metric_alerts = {
       }
     ],
   },
+  no_wallet_ingestion_alert = {
+    aggregation = "Total"
+    metric_name = "IncomingMessages"
+    description = "Payment wallet onboarding written events less than 2000 detected in the last 24h"
+    operator    = "LessThanOrEqual"
+    threshold   = 2000
+    frequency   = "PT1H"
+    window_size = "P1D"
+    dimension = [
+      {
+        name     = "EntityName"
+        operator = "Include"
+        values = [
+          "payment-wallet-evt-tx",
```

Comment:
Are we sure this is the right entity? The azure dashboard works on the event hub name while filtering per entity name, payment-wallet-ingestion-dl

Reply:
I double checked against metrics and seems you're right.
Updated values to use the eventhub name with https://github.com/pagopa/pagopa-infra/pull/2608/commits/fafd8bf4ed63bc4810d8aa055a26ae4d30182207
This configuration was taken from other alerts configuration but i think that there is a misconfiguration on other alerts